cli/admin/src/test/java/io/pravega/cli/admin/segmentstore/SegmentStoreCommandsTest.java

```diff
@@ -60,6 +64,7 @@ public void testGetSegmentInfoCommand() throws Exception {
 
     @Test
     public void testReadSegmentRangeCommand() throws Exception {
+        String filename = "./tmp/readSegmentTest.txt";
```

Comment:
Use `File.createTempFile`. You are creating a file in a `tmp` directory in the current path, which may not be writeable at all times (depending on where you run this).

Reply:
Using `File.createTempDirectory()` to create a temporary directory where the test can create, write and delete files.